aimmo-game-worker/simulation/direction.py

```diff
@@ -7,12 +7,22 @@ def __init__(self, x, y):
     def __repr__(self):
         return 'Direction(x={}, y={})'.format(self.x, self.y)
 
+    def __eq__(self, other):
+        if not isinstance(other, type(self)):
+            return NotImplemented
```

Comment:
I'd prefer not to, because a Direction and Location could technically be equal, but we want to indicate that they're two different types when compared?